enos/modules/ec2_info/main.tf

```diff
@@ -163,6 +185,125 @@ data "aws_ami" "rhel_91" {
   owners = [local.rhel_owner_id]
 }
 
+data "aws_ami" "amazon_linux_2" {
+  most_recent = true
+  for_each    = local.architectures
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-ecs-hvm-*"]
+  }
+
+  filter {
+    name   = "architecture"
+    values = [each.value]
+  }
+
+  owners = [local.amazon_owner_id]
+}
+
+data "aws_ami" "sles_12_sp4_standard" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["suse-sles-sap-12-sp4-v*-hvm-ssd-*"]
+  }
+
+  filter {
+    name   = "architecture"
+    # not available for arm64
+    values = ["x86_64"]
+  }
+
+  owners = [local.suse_owner_id]
+}
+
+data "aws_ami" "sles_12_sp5_standard" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["suse-sles-sap-12-sp5-v*-hvm-ssd-*"]
+  }
+
+  filter {
+    name   = "architecture"
+    # not available for arm64
+    values = ["x86_64"]
+  }
+
+  owners = [local.suse_owner_id]
+}
+
+data "aws_ami" "sles_15_sp4_standard" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["suse-sles-sap-15-sp4-v*-hvm-ssd-*"]
+  }
+
+  filter {
+    name   = "architecture"
+    # not available for arm64
```

Comment:
I thought for sure that they published arm64 images but then I noticed that it was only the BYOS (Bring Your Own Subscription) AMIs. Mind updating the comment that they don't currently have non-BYOS images available for arm64?